integration/operator_test.go

```diff
@@ -37,7 +37,7 @@ const (
                 GroupsClaim: ""
       Update:
         Enabled: ${GUEST_UPDATE_ENABLED}
-    Name: ci-awsop
+    Name: ci-aws-operator
```

Comment:
There is also https://github.com/giantswarm/aws-operator/blob/4d6eb492e6506841cdf9377dac17604ef24449be/.circleci/config.yml#L24. Why are there two different things and what is each used for?

Reply:
BTW I like the explicit naming way more. I am just not sure if this has implications on some AWS magic, like CF stack name length restrictions or something stupid.

Reply:
I like the more explicit name too. CF stack name length is maximum 128 chars so we're OK.

I've changed and let's see if the e2e tests are OK.

Reply:
Why do we have this in two places though? @fgimenez can you tell?

Reply:
I think its 2 different things. One is the installation name and the other if the cluster ID prefix.

I can extract to a constant because the value is the same? But IMO they aren't the same thing.

Reply:
The cluster ID is too long for the ELBs. So leaving it as is.

![screen shot 2018-03-09 at 12 37 04 pm](https://user-images.githubusercontent.com/311527/37205916-e30c3bea-2396-11e8-8e2f-497e4f71a81f.png)

Reply:
Is there a way to just hash the ELB names or cap them somehow reliably?

Reply:
It should be possible for us to use longer cluster IDs.

Reply:
> This name must be unique within your set of load balancers for the region, must have a maximum of 32 characters, must contain only alphanumeric characters or hyphens, and cannot begin or end with a hyphen.

We use the format [cluster ID]-api and [cluster ID]-ingress. This makes the ELBs easy to find in the AWS console. Hashing would be more complex and make it harder to find them.

I think this is fine as it is.

Reply:
Ok.